### PR TITLE
docs: update intro

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability.
      If this pull request addresses a security vulnerability,
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/welcome.mdx [hunk 1]
```diff
@@ -4,18 +4,26 @@ title: Welcome to Ory!
...
-- Ory Open Source offers scalable and hardened security software:
-- [Ory Kratos](https://github.com/ory/kratos)
-- [Ory Hydra](https://github.com/ory/hydra)
-- [Ory Oathkeeper](https://github.com/ory/oathkeeper)
-- [Ory Keto](https://github.com/ory/keto)
-
-Some parts of Ory are actively being developed. To learn more about roadmap and planned features head over to
-[the roadmap](./ecosystem/roadmap.md) and [project maturity](./ecosystem/versioning.md).
+Ory is a software infrastructure provider building a global zero trust network for humans, robots, devices, and logical services.
+Ory fuses open source software (Ory, Kubernetes, Crossplane, Cockroach, Linux, Istio) and open standards (Ory, OAuth2, OpenID
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:8]:
```suggestion
Ory fuses open-source software (Ory, Kubernetes, Crossplane, Cockroach, Linux, Istio) and open standards (Ory, OAuth2, OpenID
```

In `open-source software`, `open-source` is an adjective and hence should be hyphenated.

File: docs/welcome.mdx [hunk 2]
```diff
@@ -4,18 +4,26 @@ title: Welcome to Ory!
...
+Connect, MITREid, WebAuthn, TOTP) with centuries in combined open source, security, operational, and industry expertise.
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:9]:
```suggestion
Connect, MITREid, WebAuthn, TOTP) with centuries in combined open-source, security, operational, and industry expertise.
```

File: docs/welcome.mdx [hunk 3]
```diff
@@ -4,18 +4,26 @@ title: Welcome to Ory!
...
+
+[Ory Cloud](https://console.ory.sh)'s core infrastructure components are open-source to foster collaboration, reduce security
+supply-chain risk, make security accessible, and introduce the open standard for internet security. Being open source, Ory
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:12]: Why is `supply-chain` is hyphenated? It's a noun, no hyphens needed here. :)

Comment (tomekpapiernik) [inline docs/welcome.mdx:12]:
```suggestion
[Ory Cloud](https://console.ory.sh)'s core infrastructure components are open-source to foster collaboration, reduce security
supply-chain risk, make security accessible, and introduce the open standard for internet security. Being open-source, Ory
```

File: docs/welcome.mdx (hunk 1 above)
Comment (tomekpapiernik) [inline docs/welcome.mdx:8]: I might not understand the intent here, but saying that Ory fuses Ory with Ory sounds weird. Maybe there's no need to add Ory to the lists in parentheses? I think it's self-explanatory.

File: docs/welcome.mdx [hunk 4]
```diff
@@ -4,18 +4,26 @@ title: Welcome to Ory!
...
+- The Ory Identity Service is developed as an open source project called [Ory Kratos](https://github.com/ory/kratos), introducing
+  a new open standard for identity management.
+- The Ory OAuth2 & OpenID Connect Service is developed as an open source project called [Ory Hydra](https://github.com/ory/hydra),
+  implementing 15+ IETF and OpenID standards.
+- The Ory Permission Service is developed as an open source project called [Ory Keto](https://github.com/ory/keto), implementing
+  _[Zanzibar: Google’s Consistent, Global Authorization System](https://research.google/pubs/pub48190/)_ by Google.
+
+Ory develops and maintains many additional open source projects, such as the Ory Zero Trust Identity & Access Proxy
+[Ory Oathkeeper](https://github.com/ory/oathkeeper), developer tooling such as [Ory Dockertest](http://github.com/ory/dockertest)
+and language-specific libraries such as [Ory Ladon](https://github.com/ory/ladon).
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:24]: I think presenting this information as a bulleted list would be nice.

**Review by tomekpapiernik — COMMENTED** [review]
Just a couple of initial comments from my side. I'll be more than happy to have another look whenever you need me.

File: docs/welcome.mdx (hunk 4 above)
Comment (aeneasr) [reply docs/welcome.mdx:24]: Yeah I thought about that, but I did indeed wanted it to not use so much space. The rest of the document also needs to be refactored. It should basically be 20% of screen dedicated to quickly explaining what Ory is, and the rest of it to where to go next.

File: docs/welcome.mdx (hunk 1 above)
Comment (aeneasr) [reply docs/welcome.mdx]: Thanks, addressed. This was coming from grammarly

File: docs/welcome.mdx (hunk 3 above)
Comment (aeneasr) [reply docs/welcome.mdx]: Thanks, addressed. This was coming from grammarly

Comment (aeneasr) [reply docs/welcome.mdx]: Thanks, addressed. This was coming from grammarly

**Pull request merged**